### PR TITLE
Added iterationCount tests

### PR DESCRIPTION
**Pull request by kunagpal**
Takes care of #616

### CODE REVIEW

**Comment by kunagpal** [comment]
@shamasis: Okay

File: test/cli/iteration-count.test.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            try { collectionRun = JSON.parse(fs.readFileSync(collectionRunPath).toString()); }
+            catch (e) { console.error(e); }
+
+            expect(code).be(0);
+            expect(_.get(collectionRun, ITERATION_PROPERTY)).be(3);
+            done();
+        });
+    });
+
+    it('should iterate accordin to iterationCount when BOTH options are specified', function (done) {
```
Comment (shamasis) [inline test/cli/iteration-count.test.js]: typo - `accordin`

Comment (shamasis) [inline test/cli/iteration-count.test.js]: typo - accordin

Comment (kunagpal) [inline test/cli/iteration-count.test.js]: Oops.

**Pull request merged**